summary-visualiser/.gitignore

```diff
@@ -0,0 +1,2 @@
+dist
+*.html
```

Comment:
The instructions from the README generate the HTML to the root of the project. Suggest moving this content to the root `.gitignore`.